backend/src/main/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhir.java

```diff
@@ -56,12 +55,11 @@
 public class BulkUploadResultsToFhir {
 
   private static final String ALPHABET_REGEX = "^[a-zA-Z\\s]+$";
-  private static final String SNOMED_REGEX = "(^[0-9]{9}$)|(^[0-9]{15}$)";
-  private final ResultsUploaderDeviceValidationService resultsUploaderDeviceValidationService;
+  private static final String SNOMED_REGEX = "(^\\d{9}$)|(^\\d{15}$)";
```

Comment:
whats the reason for this change?

Reply:
SonarLint complaining that the digits could be replaced by `\d`, I could revert this if we prefer it to explicitly still use [0-9]